src/components/Workbench.tsx

```diff
@@ -116,23 +203,46 @@ const LoadExperience = () => {
 
   return (
     <InnerBox>
-      {expList.map((expName) => (
-        <Link
-          key={expName}
-          href={{
-            pathname: "/experience",
-            query: {
-              experienceId: expName,
-              apparatusId: "",
-              dataType: "experience",
-            },
-          }}
-        >
-          <LoadButton> {expName} </LoadButton>
-        </Link>
-      ))}
+      <table cellSpacing="0" cellPadding="0">
+        <thead>
+          <ExperienceHeader>
+            <th>EXPERIENCE NAME</th>
```

Comment:
For some reason the place of  experience name and description in the UI is sllightly different when there are no experiences in the workbench list vs when there are. Look at these two images:

![image](https://user-images.githubusercontent.com/44596199/153766155-426f3dde-8d3a-4570-b585-9921485413b2.png)

Reply:
![image](https://user-images.githubusercontent.com/44596199/153766163-c456d0b3-9c35-47cb-b3e4-dd1d97867e64.png)

Reply:
Thanks for the changes. One thing that could be improved is to have the headers and the next more to the left ( less space between exp name and the margin) and having a little bit wider items and bigger icons on the right as shown in this image :
![image](https://user-images.githubusercontent.com/44596199/153776650-a2c08a1a-475c-4469-8f31-eeebc6e1b301.png)